### ISSUE TITLE
Add support for List literal

### ISSUE THREAD

**Issue by jim-h-wang** [REGnosys/rosetta-dsl#45]
```java
post-condition: <"Ensuring that the execution attributes are correctly populated on the output.">
	if execution -> after -> execution -> party exists then
		execution -> after -> execution -> party count = 2 and
		execution -> after -> execution -> party contains partyA and
		execution -> after -> execution -> party contains partyB
	else True;
```

In the above example, we are testing whether `execution -> after -> execution -> party` contains both `partyA` and `partyB`. It would be nicer to specify the below:

```java
	if execution -> after -> execution -> party exists then
		execution -> after -> execution -> party count = 2 and
		execution -> after -> execution -> party contains [ partyA, partyB ]
```

**Comment by dhuebner** [comment]
We also add cardinality information ti the typesystem.

**Comment by dhuebner** [comment]
@jim-h-wang In context of `contains` does it mean for `[ partyA, partyB ]` all or any?

**Comment by dhuebner** [comment]
In other words is that : 
`contains partyA and  partyB` 
or 
`contains partyA or  partyB`

**Comment by dhuebner** [comment]
Sehe auch  #67

**Comment by dhuebner** [comment]
@jim-h-wang 
I've added support for list literal to the RosettaLiteral. Means it can appear anyway where RosettaExpression can be used. On the generator side a can only confirm that it would work with your example above (in conditions) but I cant check all the possible occurrences. Especially it's not clear which library functions should/can support it. Hence, we need to define where a ListLiteral can/should be used. 

Tested:

```
if execution -> after -> execution -> party exists then
	[partyA, partyB] count = 2 and party contains [partyA, partyB]
```
